guides/doc-Planning_for_Project/master.adoc

```diff
@@ -14,29 +14,36 @@ ifdef::satellite[]
 include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[leveloffset=+1]
 endif::[]
 
-[[part-Architecture]]
-= {Project} Architecture
+[id="Project_Overview_and_Concepts_{context}"]
+= {Project} Overview and Concepts
 
-include::topics/Introduction.adoc[]
+{ProjectName} is a centralized tool for provisioning, remote management, and monitoring of multiple {EL} deployments.
+With {Project}, you can deploy, configure, and maintain your systems across physical, virtual, and cloud environments.
 
-include::topics/Capsule_Server_Overview.adoc[]
+include::common/assembly_content-and-patch-management-with-project-name.adoc[leveloffset=+1]
 
-include::topics/Org_Loc_and_Life_Cycle_Environments.adoc[]
+include::common/assembly_subscription-management-with-projectname.adoc[leveloffset=+1]
 
-include::topics/Hosts.adoc[]
+include::common/assembly_provisioning-management-with-projectname.adoc[leveloffset=+1]
 
-include::topics/Provisioning_Concepts.adoc[]
+include::common/assembly_major-project-components.adoc[leveloffset=+1]
 
-ifdef::satellite[]
-include::topics/Content_Delivery_Network.adoc[]
-endif::[]
+include::common/assembly_tools-for-administration-of-projectname.adoc[leveloffset=+1]
+
+include::common/assembly_supported-usage-and-versions-of-project-components.adoc[leveloffset=+1]
 
 [[part-Deployment_Planning]]
 = {Project} Deployment Planning
```

Comment:
Please move to a separate file in `guides/common/modules`.

Reply:
The Deployment Planning part of the guide is out of scope of this PR. (But definitely a to-do, I plan to review it after I'm done with the Overview part).